V/VerizonEdgecastTokenBuilder/build_tarballs.jl

```diff
@@ -0,0 +1,44 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "VerizonEdgecastTokenBuilder"
```

Comment:
Why this long name, with trailing "builder"? My understanding is that the project is called just "ectoken", is that not correct?